### DISCUSSION TITLE
[Kavita] Feature — Option to display odd/even pages on the left or right

### DISCUSSION THREAD

**Discussion by audreytoskin** [Kareadita/Kavita#2750] [Ideas]
### Idea Description

Printed books may or may not include intentionally blank pages, or content on the inside cover, etc, and scanned ebooks may or may not include all of those "pages"... which can throw off multi-paginated layouts.

This mostly matters for comics when viewing two-page spreads. We need an option, easily toggleable for different books, to put odd or even pages on the left or right...

An example of what I'm talking about, from another ereader app: Komga's UI calls it "Double pages" versus "Double pages (no cover)" layout.

A demonstration website is available at: https://demo.komga.org/

You can log in using the following credentials:

    Login: demo@komga.org
    Password: komga-demo

Toggle between page layouts by pressing D.

### Idea Category

API

### Before submitting

- [x] I've already searched for existing ideas before posting.

**Comment by scare376** [comment]
What reader are you requesting this for? This is already a thing with the image reader.

**Reply by scare376** [reply]
Kavita has 3 different readers. The "archive" (which handles images only), the epub, and the PDF reader.

**Comment by audreytoskin** [comment]
(Sorry, I should clarify a point that I somehow truncated in my original post)
In Komga, the feature is called **"Double pages" and "Double pages (no cover)" layout** . What I was trying to request wasn't merely about allowing double-page view at all, or supporting "Manga mode"... What I would like, when reading a comic (or other ebook) in double-page view, is to be able to toggle whether the odd-number pages:

- go on the right / 'page 1' is treated like a book cover (not part of a two-page spread), and therefore the first two-page spread is pages 2 and 3, then pages 4 and 5, etc --- or,
- odd-number pages go on the left / the first page is part of a two-page spread, so the spreads are pages 1 and 2, then 3 and 4, etc.

In Kavita 0.7.14, as far as I can tell, the only double-page view options are "Double" and "Double (Manga)", which are more about changing whether pages flow left-to-right or right-to-left, as opposed to changing the way two-page spreads are grouped.

**Comment by scare376** [comment]
This is a duplicate of #2660.
